ext/libxml/tests/libxml_get_external_entity_loader_error_callback_name.phpt

```diff
@@ -17,8 +16,9 @@ DTD;
 
 class Handler {
 	public function handle($public, $system, $context) {
-	    $file = __DIR__ . '/db.dba';
-		return dba_open($file, 'n');
+        $descriptors = [["pty"], ["pty"], ["pty"], ["pipe", "w"]];
+        $pipes = [];
+        return proc_open('echo "foo";', $descriptors, $pipes);
```

Comment:
I don't know if it makes sense to create a resource in `ext/zend_test` for those sorts of purposes?

Reply:
I agree with Gina, I'd use zend_test for this unless that would be too cumbersome.

Reply:
Yes, a dedicated test-only resource is our last resort as soon as we run out of regular non-stream resources. Until 9.0, we are fine with `proc_open()`, so I'd postpone this hassle until then, if you are ok with it.